plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleDependencyResolutionHelper.java

```diff
@@ -260,4 +345,45 @@ private static void printDependencyMap(Map<DependencyDescriptor, Set<DependencyD
         }
         return value;
     }
+
+    // Translate the different Gradle dependency scopes in to values that map to ShrinkWrap's scope type.
+    // c.f.: https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management
+    private static final Map<String, String> REMAPPED_SCOPES = new HashMap<>();
+
+    static {
```

Comment:
I had to rework this class to take in to account all possible "known" configurations (a.k.a., dependency scopes) for Gradle 5 based projects. In the above code, there is a *fallback* which translates unknown configurations (a.k.a., user defined configurations) in to the closest "known" configuration. I have tested this behavior as well.